server/routes/dashBoard.js

```diff
@@ -0,0 +1,251 @@
+const express = require("express");
+const router = express.Router();
+const { updateData, decodeToken } = require("../util/util");
+
+// Models;
+const { Task, Column, Dashboard } = require("../models/Dashboard");
+
+//@CreateBoard
+router.post("/getDashboard", async (req, res) => {
+  const { token } = req.body;
+  let userId = await decodeToken(token);
+
+  try {
+    let dashboard = await Dashboard.findOne({ user: userId });
+    res.send(dashboard);
+  } catch (err) {
+    res.status(404).json({ error: "Dashboard does not exist" });
+  }
+});
+
+//Add Dashboard @in progress need to update UserId
+router.post("/addDashBoard", async (req, res) => {
```

Comment:
Similar naming issue as above

Reply:
thank you for reviewing!
- changed route name to /dashboards/dashboard/ from /dashboard/addDashboard